### PR TITLE
Implement Account Deactivation and Activation

### PR DESCRIPTION
**Pull request by samolubode**
This PR:
- Addresses #29
- Deactivates a user's account for 30days of inactivity upon Profile deletion request
- Activate a user if they login within 30 days of deactivate.
(For both normal and social users)

The eventual deletion of an account is attached to the Profile route i.e whenever a profile is viewed, the controller checks for and deletes accounts that are due to deletion.

### CODE REVIEW

File: models/postModel.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+const db = require('./index')
```
Comment (ademesojosiah) [inline models/postModel.js]: whats the db here for

File: models/index.js [hunk 2]
```diff
@@ -24,11 +24,11 @@ const logger = require("./../utils/logger");
 const sequelize = new Sequelize(DB_NAME, DB_USER, DB_PASSWORD, {
   host: DB_HOST,
   dialect: DB_DIALECT,
-  dialectOptions: {
-    ssl: {
-      SSL,
-    },
-  },
+  // dialectOptions: {
```
Comment (ademesojosiah) [inline models/index.js]: un comment the dialectOptions my  oga

File: models/postModel.js (hunk 1 above)
Comment (samolubode) [reply models/postModel.js]: I guess I was trying something out

File: models/index.js (hunk 2 above)
Comment (samolubode) [reply models/index.js]: Done, boss

**Pull request merged**